docs/sources/fundamentals/intro-to-prometheus/index.md

```diff
@@ -0,0 +1,110 @@
+---
+aliases:
+  - ../basics/timeseries/
+description: Introduction to Prometheus
+keywords:
+  - grafana
+  - intro
+  - Prometheus
+  - metrics
+  - time series
+title: What is Prometheus?
+weight: 300
+---
+
+# What is Prometheus?
+
+Observability focuses on understanding the internal state of systems based on the data they produce, which helps determine if the infrastructure is operating as expected. Prometheus is a core technology for system observability. However, Prometheus can be a confusing term because it is used in different contexts. Understanding Prometheus basics, why it’s valuable, and how people use it in practice will help you use Grafana.
+
+Prometheus was started in 2012 at SoundCloud, who found existing technologies such as Graphite insufficient for their observability needs. Prometheus has a robust data model and query language, both of which are addressed in the following sections, and is simple and scalable. In 2018, Prometheus graduated from Cloud Native Computing Foundation (CNCF) incubation, and today has a thriving community.
+
+## Prometheus as data
+
+The following panel in a Grafana dashboard shows how much disk bandwidth on a Mac laptop is being used. The green line represents disk `reads`, and the yellow line represents `writes`.
+
+{{< figure src="/media/docs/grafana/intro-prometheus/disk-io.png" max-width="750px" caption="Disk I/O dashboard" >}}
+
+Data like these form _time series_. The X-axis is a moment in time, and the Y-axis is a number or measurement, for example, 5 megabytes per second. This kind of time series data appears everywhere in systems monitoring, but also in places like seasonal temperature charts and stock prices. This data is simply some measurement (such as the Tesla stock price, or Disk I/O) through a series of time instants.
+
+Prometheus is a technology used to collect and store time series data. Time series are fundamental to Prometheus; its [data model](https://prometheus.io/docs/concepts/data_model/) is arranged into:
```

Comment:
We attempted to cover this much earlier in sections.  Does this get at it?

> Prometheus is a core technology for system observability. However, Prometheus can be a confusing term because it is used in different contexts. Understanding Prometheus basics, why it’s valuable, and how people use it in practice will help you use Grafana.

Reply:
I prefer how it reads now cause we're defining time series by using an example. then we can talk about prometheus and the data model.